src/newtondynamics_physics/ospnewton.h

```diff
@@ -70,34 +83,28 @@ struct ACtxNwtWorld
     };
 
     ACtxNwtWorld(int threadCount)
-     : m_nwtWorld(NewtonCreate())
-     , m_perThread(threadCount)
+     : m_world(NewtonCreate())
```

Comment:
I guess either we should fix the thing that the CodeScanner is complaining about, or we should turn that warning off.